src/architecture/principles.md

```diff
@@ -0,0 +1,287 @@
+> the Internet community delegates authority, via the IANA URI scheme registry and the
+> DNS, over a set of URIs with a common prefix to one particular owner. One consequence
+> of this approach is the Web's heavy reliance on the central DNS registry.(…)
+>
+> URI owners are responsible for avoiding the assignment of equivalent URIs to multiple
+> resources. Thus, if a URI scheme specification does provide for the delegation of
+> individual or organized sets of URIs, it should take pains to ensure that ownership
+> ultimately resides in the hands of a single social entity. Allowing multiple owners
+> increases the likelihood of URI collisions.
+>
+> URI owners may organize or deploy infrastruture [sic] to ensure that representations of
+> associated resources are available and, where appropriate, interaction with the resource
+> is possible through the exchange of representations. There are social expectations for
+> responsible representation management (§3.5) by URI owners. Additional social
+> implications of URI ownership are not discussed here.
+
+This notion of address or name ownership is
+[pervasive across architectural documents](https://www.w3.org/DesignIssues/). This passage
+from an interview of TimBL
+([Philosophical Engineering and Ownerhip of URIs](https://www.w3.org/DesignIssues/PhilosophicalEngineering.html)) is explicit:
+> **Alexandre Monnin**: Regarding names and URIs, a URI is not precisely a philosophical
+> concept, it's an artifiact [sic]. So you can own a URI while you cannot own a philosophical
+> name. The difference is entirely in this respect.\
+> **Tim Berners-Lee**: For your definition of a philosophical name, you cannot own it.
+> Maybe in your world, in your philosophy, you don't deal with names that are owned, but
+> in the world we're talking about, names are owned.
+
+This expectation of delegated naming authority was so strong among early Web architects
+that the development of naming conventions in HTTP space (eg. `robots.txt`, `favicon.ico`,
+all the `.well-known` paths) is described as "*expropriation*" in the
+[Web Architecture](https://www.w3.org/TR/webarch/) and the W3C's Technical Architecture
+Group (TAG) issue on the topic stated that it "breaks the web".
+
+Federated models only have weak capture-resistance because the federated entities can always
+concede power (precisely because they have ownership) but lack established means to
+support collective organization. As a result, any power imbalance will likely become hard
+to dislodge. A good example is search: as a publisher (the owner of delegated authority
+over your domain) you can cede the rights to index your content but you can't have a voice
+in what is done with the indexed content (individual opt out is not an option). This was
+fine when you could barter content for links, but once search power consolidated, the
+terms of trade deteriorated with no immediate recourse.
```

Comment:
:thought_balloon: interestingly, history does not repeat, but rhymes :) 
 [Federated IPNIs](https://github.com/ipni) will fight the same losing battle, and will have to concede power to support arbitrary transports (https://github.com/ipfs/specs/issues/377).